pages/linux/bpytop.md

```diff
@@ -0,0 +1,28 @@
+# bpytop
+
+> Display dynamic real-time information about running processes with graphs. Similar to `gtop` and `htop`.
+> More at <https://github.com/aristocratos/bpytop>.
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
> More information: <https://github.com/aristocratos/bpytop>.
```

Reply:
Sure.